.github/workflows/triage.yml

```diff
@@ -64,6 +64,7 @@ jobs:
               path: Formula/.+@.+
               except:
                 - Formula/bash-completion@2.rb
+                - Formula/openssl@3.0.rb
```

Comment:
I'm starting to reconsider if this should actually be declared as https://github.com/Homebrew/homebrew-core/labels/legacy. I mean, it's not really old, but `python@3.10` is not old either.

Reply:
Dropped this.